docker/build_docker_images.sh

```diff
@@ -0,0 +1,19 @@
+for FRAMEWORK in "jax" "pytorch" "joint"
```

Comment:
nit: can you rename this to build_all_dev_docker_images.sh?

Reply:
Sorry, I hadn't finalized this yet. I added a flag to specify the branch. Is that ok w you?